### ISSUE TITLE
DockerHubPullSuite is off on Windows

### ISSUE THREAD

**Issue by jhowardmsft** [docker/docker#16090]
As it was breaking Windows CI after #16039 was merged. Was turned off temporarily through PR #16087. @icecrime

**Comment by LK4D4** [comment]
@jhowardmsft I think this should be addressed. Do you know who can do that? I can try, but it will be painful to test through CI.

**Comment by jhowardmsft** [comment]
Agreed. On both. I've assigned myself, not sure when I can get to it though, or quite what it will entail.

**Comment by vdemeester** [comment]
@jhowardmsft what should we do about that one ? /cc @thaJeztah @johnstep

**Comment by jhowardmsft** [comment]
It's still on my list, but haven't had time to look to see what it would take. I think there's some CI infrastructure which would need standing up to get it to work.
